features/configuration/run_all_when_everything_filtered.feature

```diff
@@ -1,78 +0,0 @@
-Feature: run all when everything filtered
```

Comment:
I know why you removed this but this is essentially "disappearing" a feature without a major release (from a documentation pov, which is why I wonder if we should be more verbose with our deprecation)